### PR TITLE
Update test .gitignore

### PR DESCRIPTION
**Pull request by mklim**
`testing/dart/android/` and `testing/dart/ios` both look like generated
directories created after each `run_tests` execution. Also updates that
script to use `flutter pub get`.

### CODE REVIEW

File: testing/run_tests.sh [hunk 1]
```diff
@@ -6,7 +6,7 @@ out/host_debug_unopt/fml_unittests
 out/host_debug_unopt/synchronization_unittests
 
 pushd flutter/testing/dart
-pub get
+flutter pub get
```
Comment (jason-simmons) [inline testing/run_tests.sh]: What is the reason for this change?

I'd like to avoid having the engine tree depend on the framework tree

Comment (mklim) [reply testing/run_tests.sh]: Discussed offline, removed.
